RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.cc

```diff
@@ -250,7 +250,6 @@ double EcalUncalibRecHitWorkerMultiFit::timeCorrection(
 }
 
 
-
 bool
 EcalUncalibRecHitWorkerMultiFit::run( const edm::Event & evt,
                 const EcalDigiCollection::const_iterator & itdg,
```

Comment:
Sorry, but I refuse to write explicitly inefficient code, and I hope nobody would ever approve such change to enter the release.

If having both versions is really such a problem, you should convince the ECAL software experts to migrate all other implementation to the more efficient interface, and then frop the less efficient one.